person.rb

```diff
@@ -5,14 +5,15 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
     super()
   end
 
   def correct_name
     name
   end
 
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
   attr_reader :id
 
   private
```

Comment:
- [ ] A method for adding rentals should be present. The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association.